server/integration/jgroups/src/main/resources/subsystem-templates/infinispan-jgroups.xml

```diff
@@ -19,8 +19,8 @@
                 <protocol type="UNICAST3" />
                 <protocol type="pbcast.STABLE" />
                 <protocol type="pbcast.GMS" />
-                <protocol type="UFC" />
-                <protocol type="MFC" />
+                <protocol type="UFC_NB" />
```

Comment:
Are these changes related to this JIRA?

Reply:
Yes, removing blocking in JGroups is another step in the async crusade.